Alloy/Renderers/BoundingBoxesRenderer.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  BoundingBoxesRenderer.swift
+//  Alloy
+//
+//  Created by Eugene Bokhan on 22/04/2019.
+//
+
+import Metal
+import simd
+
+@available(iOS 11.3, tvOS 11.3, macOS 10.13, *)
+final public class BoundingBoxesRenderer {
+
+    public enum Errors: Error {
+        case wrongRenderTargetTextureUsage
+        case missingRenderTarget
+    }
+
+    // MARK: - Properties
+
+    /// Rectrangles in a normalized coodrinate system describing bounding boxes.
+    public var normalizedRects: [CGRect] = []
+    /// Rrefered fill color of the bounding boxes. Red is default.
+    public var color: vector_float4 = .init(1, 0, 0, 1) {
+        didSet {
+            self.linesRenderer.color = self.color
+        }
+    }
+    /// Prefered line width of the bounding boxes in pixels. 20 is default.
+    public var lineWidth: Int = 20
+    /// Size of the render target texture.
+    public var renderTargetSize: MTLSize = .zero
```

Comment:
why do we need to know it?

Reply:
`renderTargetSize` is used to calculate vertical & horizontal widths of line components of the bbox.